src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,25 @@
+/* eslint-disable implicit-arrow-linebreak */
```

Comment:
Please remove this eslint disable rule and fix the issues it's trying to suppress. Disabling rules should be used as a last resort.